### ISSUE TITLE
NotFittedError using scikit-learn validation

### ISSUE THREAD

**Issue by afikbar** [catboost/catboost#1783]
Problem:

The bug raised when used CatBoostClassifier (prefitted) with CalibratedClassifierCV.
Apparently, CatBoostClassifier doesn't include "post fit" attributes (in __dict__, and hence in vars(model)), but rather defines a property:

```python
@property
def learning_rate_(self):
    return getattr(self, '_learning_rate') if self.is_fitted() else None
```

These properties are not accessible through vars built-in, and thus, 'sklearn.utils.validation.check_is_fitted' raises NotFittedError.

Example:
```python
from catboost import CatBoostClassifier
from sklearn.calibration import CalibratedClassifierCV

model = CatBoostClassifier()
model.fit(X_train, y_train)

cc_model = CalibratedClassifierCV(model, cv='prefit', method='isotonic')
model = cc_model.fit(X_validation, y_validation)
```
catboost version: 0.26
Operating System: Windows 10

**Comment by afikbar** [comment]
Suggested workaround:

```python

from catboost import CatBoostClassifier
from sklearn.calibration import CalibratedClassifierCV

model = CatBoostClassifier()
model.fit(X_train, y_train)

model.__dict__.update({v: getattr(model, v) for v in dir(model) if v.endswith("_") and not v.startswith("__")})
cc_model = CalibratedClassifierCV(model, cv='prefit', method='isotonic')
model = cc_model.fit(X_validation, y_validation)
```

**Comment by kizill** [comment]
Fixed in 0.26.1
